draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1531,23 +1531,22 @@ one of the three mechanisms listed above.
 
 ## Multiple SCTs {#multiple-scts}
 
-TLS servers SHOULD send SCTs from multiple logs in case one or more logs are not
-acceptable to the TLS client (for example, if a log has been struck off for
-misbehavior, has had a key compromise, or is not known to the TLS client). For
-example:
+CT-using TLS servers SHOULD send SCTs from multiple logs, because:
+
+* One or more logs may not have become acceptable to all CT-using TLS clients.
```

Comment:
Nit: replace "not have become acceptable" with simply "not be acceptable". That encapsulates cases where a log was struck off, etc.

Reply:
A log being struck off is the third bullet point.  I wanted to avoid repetition and let each bullet point be distinct.